invenio_rdm_records/utils.py

```diff
@@ -23,7 +23,7 @@ def get_or_create_user(email):
             username=email.split("@")[0],
             password=hash_password("123456"),
             active=True,
-            preferences=dict(visibility="public", email_visibility="public"),
+            preferences=dict(visibility="public", email_visibility="public", preferences_locale='ch'),
```

Comment:
you could eventually omit this one, it is used in the fixtures: it will use the default.
```suggestion
            preferences=dict(visibility="public", email_visibility="public", locale="en_GB"),
```